cmd/finch-daemon/main.go

```diff
@@ -215,8 +222,20 @@ func newRouter(options *DaemonOptions, logger *flog.Logrus) (http.Handler, error
 		return nil, err
 	}
 
-	opts := createRouterOptions(conf, clientWrapper, ncWrapper, logger)
-	return router.New(opts), nil
+	var regoFilePath string
+	if options.enableOPAMiddleware {
+		regoFilePath, err = checkRegoFileValidity(options, logger)
```

Comment:
Are there any other policy language that OPA may support?

Reply:
Explained in a comment above